dataflow/tests/reach-definition/Test.java

```diff
@@ -0,0 +1,14 @@
+// This file may not be renamed; it has to have the same filename as ../issue3447/Test.java .
```

Comment:
Maybe add some extra test files? I think we also need an example under `dataflow/manual/examples`.

Reply:
Ok, I think I will add one more test file in this pr and a new test file in my next pr when I remove StringConcatenateAssignmentNode.